README.md

```diff
@@ -139,29 +177,46 @@ the Web (HTTP & FTP)
 * [TeraCopy](https://www.codesector.com/downloads): Copy your files faster and more securely
 * [tree](http://mama.indstate.edu/users/ice/tree/): 'tree' command for linux
 * [WinDirStat](https://windirstat.net/): Disk usage statistics viewer and cleanup tool for Windows
+* [WizTree](https://antibody-software.com/web/software/software/wiztree-finds-the-files-and-folders-using-the-most-disk-space-on-your-hard-drive/): Finds the files and folders using the most disk space on your hard drive
```

Comment:
Does WizTree have any advantages over WinDirStat? At quick glance it looks like the same thing as WinDirStat. (If possible I'd like to avoid promoting non-free software

Reply:
WizTree reads from the Master File Table, similar to Everything (another tool on this list), so the graphs are made much quicker. I didn't see #7 until after I made the PR, however, so I thought you were still accepting non-free/closed source software. It can be excluded. I agree, the list should consist primarily of open source solutions, and maybe have a separate list for the rest.

Reply:
Okay, if there's an advantage that's fine for now, we'll figure out the categorization of licenses later on

Reply:
Yeah, I prefer free software also but WizTree is ridiculously faster than WinDirStat. I just ran a test on my laptop with a midrange SSD, and WinDirStat took 4 min 4 sec to scan my C: drive with 111 GB used, and WizTree took... 6 seconds.